frontend/src/CreateNew.jsx

```diff
@@ -1,40 +1,51 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-unused-vars */
 import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
 import axios from 'axios';
 import CreateForm from './CreateForm';
 import styles from './Login.module.css';
 
 function CreateNew() {
   const [user, setUser] = useState([]);
+  const history = useHistory();
 
+  // eslint-disable-next-line no-shadow
   async function makePostCall(user) {
     try {
       const response = await axios.post('http://localhost:5000/signup', user);
-      return response;
+      if (response.status === 201) {
+        return true;
+      }
+      return false;
```

Comment:
This should catch a 409 - email already exist - in an else. If it throws something other than 409 it would be a backend problem

Reply:
This would be good to add.

Reply:
That's true, I didn't consider that it might throw something other than a 201 and 409. I'll make that change for the other responses as well.